source/manual/pagerduty.html.md

```diff
@@ -1,13 +1,29 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: PagerDuty drill
-parent: "/manual.html"
+title: PagerDuty
+section: 2nd line
+type: learn
 layout: manual_layout
-section: Icinga alerts
-last_reviewed_on: 2019-07-18
-review_in: 6 months
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 3 months
 ---
 
+Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming
+unavailable or large quantities of error pages being served. We use [PagerDuty](https://governmentdigitalservice.pagerduty.com)
+to escalate to these people (in order), mirroring the out-of-hours escalation order:
```

Comment:
The same configuration is used in hours and out of hours, there's only a few differences in the thresholds for different checks, and normally, who's on the schedule. So I'd perhaps tweak the bit around "mirroring the out-of-hours escalation order" as I don't consider the in-hours and out-of-hours process as different.